samples/web/blazor-server-application/Core_8/Server/CommandMessageHandler.cs

```diff
@@ -0,0 +1,27 @@
+﻿using NServiceBus;
+using NServiceBus.Logging;
+
+#region Handler
+
+public class CommandMessageHandler :
+    IHandleMessages<Command>
+{
+    static ILog log = LogManager.GetLogger<CommandMessageHandler>();
+
+    public Task Handle(Command message, IMessageHandlerContext context)
+    {
+        log.Info("Hello from CommandMessageHandler");
+        Task reply;
+        if (message.Id % 2 == 0)
+        {
+            reply = context.Reply(ErrorCodes.Fail);
+        }
+        else
+        {
+            reply = context.Reply(ErrorCodes.None);
+        }
```

Comment:
This could be simpler and more expressive in what gets logged

```suggestion
        if (message.Id % 2 == 0)
        {
            log.Warn("Message ID is odd, returning a failure.");
            return context.Reply(ErrorCodes.Fail);
        }
        
        log.Info("Message ID is even, returning success.");
        return context.Reply(ErrorCodes.None);
```

Reply:
Yeah, I agree with you.  I was just trying to follow the POA of the issue https://github.com/Particular/docs.particular.net/issues/5735 to mirror the [asp.net core sample](https://github.com/Particular/docs.particular.net/blob/master/samples/web/asp-web-application/Core_8/Server/CommandMessageHandler.cs#L14-L24)

Reply:
Ah, Ok. Then ignore my comment.